rasa/shared/core/slots.py

```diff
@@ -298,8 +298,15 @@ def add_default_value(self) -> None:
             )
 
     def persistence_info(self) -> Dict[Text, Any]:
+        """Returns serialized slot."""
         d = super().persistence_info()
-        d["values"] = self.values
+        d["values"] = [
+            value
+            for value in self.values
+            # Don't add default slot when persisting it.
+            # We'll re-add it on the fly when creating the domain.
+            if value != rasa.shared.core.constants.DEFAULT_CATEGORICAL_SLOT_VALUE
```

Comment:
Isn't this handed by the change in `_slot_definitions`?

Reply:
No. The `default_slot_value` is a **value** and not an entire slot. Means we can't exclude the `entire` slot, however, we can exclude the `default` and then re-add it later.